WireUI/Sources/WireMainNavigation/Resources/Localizable.xcstrings

```diff
@@ -374,425 +374,1457 @@
       }
     },
     "tabBar.contacts.description" : {
-
-    },
-    "tabBar.contacts.hint" : {
-
-    },
-    "tabBar.contacts.title" : {
-      "localizations" : {
-        "en" : {
-          "stringUnit" : {
-            "state" : "translated",
-            "value" : "Contacts"
-          }
-        }
-      }
-    },
-    "tabBar.conversations.description" : {
       "localizations" : {
         "ar" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
```

Comment:
@caldrian Is this expected or do you think something has gone wrong with localization for this entry?